app/src/main/java/org/oppia/app/topic/revision/TopicRevisionFragment.kt

```diff
@@ -13,6 +13,18 @@ import javax.inject.Inject
 
 /** Fragment that card for topic revision. */
 class TopicRevisionFragment : InjectableFragment(), RevisionSubtopicSelector {
+  companion object {
+    /** Returns a new [TopicRevisionFragment] */
```

Comment:
Add full-stop at the end of the sentence `/** Returns a new [TopicRevisionFragment]. */`

Reply:
Done